components/BaseComponents.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/native';
+import Colors from '../assets/Colors';
+
+// Text Components
+export const Title = styled.Text`
+  font-family: poppins-medium;
+  font-size: 20px;
+  line-height: 30px;
+  color: ${props => (props.color ? props.color : Colors.black)};
```

Comment:
Can we change this everywhere to `${props => props.color || Colors.black}`? It's just a cleaner way to do this suggested by @emilyzhong